internal/services/securitycenter/security_center_server_vulnerability_assessment_hybrid_vm_resource.go

```diff
@@ -0,0 +1,146 @@
+package securitycenter
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/security/mgmt/v3.0/security"
+	"github.com/hashicorp/terraform-provider-azurerm/helpers/tf"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	computeParse "github.com/hashicorp/terraform-provider-azurerm/internal/services/compute/parse"
+	computeValidate "github.com/hashicorp/terraform-provider-azurerm/internal/services/compute/validate"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+const (
+	hybridProvider = "Microsoft.HybridCompute"
+	hybridType     = "machines"
+)
+
+func resourceServerVulnerabilityAssessmentHybridVM() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Create: resourceServerVulnerabilityAssessmentHybridVMCreate,
+		Read:   resourceServerVulnerabilityAssessmentHybridVMRead,
+		Delete: resourceServerVulnerabilityAssessmentHybridVMDelete,
+
+		// TODO: replace this with an importer which validates the ID during import
+		Importer: &schema.ResourceImporter{
+			StateContext: schema.ImportStatePassthroughContext,
+		},
```

Comment:
could we add a custom importer which validates the Resource ID and confirms that this is a Hybrid VM? I can't make a suggestion for some reason, but that'd be `pluginsdk.ImporterValidatingResourceId`